templates/SilverStripe/ElementalBlocks/FileBlock.ss

```diff
@@ -0,0 +1,7 @@
+<% if $Title && $ShowTitle %>
+    <h2 class="element__title">$Title</h2>
+<% end_if %>
+
+<div class="element__file">
+    $File
+</div>
```

Comment:
As discussed in the content-blocks slack channel, I feel the css classes would be more useful to css developers if they followed a pattern more like this:
  
`fileelement`
`fileelement_title`
`fileelement_text`
etc 

For the most part elements aren't blocks in the BEM sense, but namespaces. When we style them, they normally have very little to do with each other. They exist in seperate scss/less/css files and are treated as seperate components. It is worth keeping the word element in there for namespace reasons, but its less important than the unique name of component that is being styled.

Reply:
If we are to follow whats already in core and BEM standards it would be better to stick with .file-element__text or .file-block__title . It helps to give people a reference to the standards rather than creating our own and it is already in use in core. It actually helps to differentiate from bootstrap class names as a well as all the default reasons for using the standard.

Reply:
Reference issue to update: #12